### ISSUE TITLE
Override cTrader default host

### ISSUE THREAD

**Issue by desoss** [Reiryoku-Technologies/Mida-cTrader#33]
According to the CTrader [docs](https://help.ctrader.com/open-api/proxies-endpoints/#proxies), Mida-cTrader is using the old proxy to connect to CTrader.
In fact, in _CTraderApplication.ts_ the default host and port are these:
` 
this.#demoConnection = new CTraderConnection({ host: "demo.ctraderapi.com", port: 5035, });
this.#liveConnection = new CTraderConnection({ host: "live.ctraderapi.com", port: 5035, });
`

Can we have a way to override the default configured hosts?

By the way, using the old proxy in Oregon takes 6 seconds to log me in 😅 
While from Dublin, Frankfurt and Milan around 1.5 seconds.
Are there other ways to improve the performances?

**Comment by Vasile-Peste** [comment]
I will take care of this!

**Comment by desoss** [comment]
Do you have an idea of the effort? Can you please give me an approximate date for the possibility to configure the host?

**Comment by Vasile-Peste** [comment]
> Do you have an idea of the effort? Can you please give me an approximate date for the possibility to configure the host?

We have just released [Mida cTrader 5.3](https://github.com/Reiryoku-Technologies/Mida-cTrader/releases/tag/v5.3.0), can you please update and test?

The new parameters are not documented yet but this is an example.

```javascript
const myAccount = await login("cTrader", {
    clientId: "***",
    clientSecret: "***",
    accessToken: "***",
    cTraderBrokerAccountId: "***",
    demoProxy: "demo-eu-1.ctraderapi.com",
    liveProxy: "live-eu-1.ctraderapi.com",
});
```

The new parameters are optional, so if you don't pass them you are going to use the default old proxies.
Furthermore, if the account is demo you can just omit the live proxy parameter, same for live accounts: you can omit the demo proxy.

Reference https://help.ctrader.com/open-api/proxies-endpoints/
